e2e_test/subscription/main.py

```diff
@@ -304,6 +304,27 @@ def test_cursor_fetch_n():
     check_rows_data([10,100],row[3],3)
     drop_table_subscription()
 
+def test_rebuild_table():
+    print(f"test_rebuild_table")
+    create_table_subscription()
+    conn = psycopg2.connect(
+        host="localhost",
+        port="4566",
+        user="root",
+        database="dev"
+    )
+
+    execute_insert("declare cur subscription cursor for sub2",conn)
+    execute_insert("insert into t2 values(1,1)",conn)
```

Comment:
We may manually trigger ad-hoc recovery, or run a sink-into-table, to cover the error case.

Reply:
add sink into table in create_table_and_subscription.slt